packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -313,7 +341,156 @@ const ToggleMenuDrawer = React.forwardRef(
                                     )}
                                 </MobileDrawer.Body>
                             )}
-                            {!is_appstore && (
+                            {is_pre_appstore && (
```

Comment:
The platform switcher should be available in nav menu when  
`is_pre_appstore && 
( window.location.pathname === routes.trading_hub ||
 window.location.pathname === routes.cashier ||
window.location.pathname === route.account)

Reply:
Alrightss